Sources/IBLinterFrontend/Commands/ValidateCommand.swift

```diff
@@ -26,11 +26,14 @@ struct ValidateCommand: CommandProtocol {
             fatalError("\(workDirectoryString) is not directory.")
         }
 
-        let config = Config(options: options) ?? Config.default
+        var config = Config(options: options) ?? Config.default
         if config.disableWhileBuildingForIB &&
             ProcessInfo.processInfo.compiledForInterfaceBuilder {
             return .success(())
         }
+        if config.included.isEmpty {
+            config.included = options.included
```

Comment:
because all you do is modified config object from information defined by options